src/components/Card.js

```diff
@@ -1,9 +1,23 @@
-class Card {
-  constructor(data, cardSelector, _handleImageClick) {
-    this._name = data.name;
-    this._link = data.link;
+export default class Card {
+  constructor(
+    { name, link, _id, isLiked },
+    cardSelector,
+    handleCardClick,
+    handleTrashClick,
+    handleLikeClick
+  ) {
+    this._name = name;
+    this._link = link;
     this._cardSelector = cardSelector;
-    this._handleImageClick = _handleImageClick;
+    this._handleCardClick = handleCardClick;
+    this._handleTrashClick = handleTrashClick;
+    this._handleLikeClick = handleLikeClick;
+    this._cardId = _id;
+    this._isLiked = isLiked;
+    this._cardElement = document
```

Comment:
Needs correction: you should create the card element in the beginning of ` getView` method, not here